webpack/webpack.config.js

```diff
@@ -1,31 +1,28 @@
-const path = require('path');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-
-const HtmlWebpackPluginConfig = new HtmlWebpackPlugin({
-  template: './src/index.html',
-  filename: 'index.html',
-  inject: 'body'
-})
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
 
+console.log(path.resolve(__dirname, '../src/index.html'))
 module.exports = {
-  entry: './src/app/main.js',
-  output: {
-    path: path.resolve(__dirname, 'dist'),
-    filename: 'bundle.js'
-  },
+  entry: path.resolve(__dirname, '../src/app/main.js'),
+  output: { path: path.resolve(__dirname, '../dist'), filename: 'bundle.js' },
   module: {
     loaders: [
       {
-        test: /\.js$/,
+        test: /(\.js)|(\.jsx)$/,
```

Comment:
since you used `.jsx` testcase as separate loader in you config, i combined it to one using above regex, but i dont why tou want to support `.jsx` files. `.js` files are sufficient to write React code